docs/guides/SimpleStartupGuide.md

```diff
@@ -0,0 +1,396 @@
+# USB Camera Device Service Simple Startup Guide
+
+## Contents
```

Comment:
A lot of this stuff is duplicated in both `setup.md` and the advanced guide. I think the pre-reqs can all be done in just one file, and then it branches off from there.

Reply:
ok sounds good. I will change it.